### ISSUE TITLE
Cobblestone Generator - BlockFromToEvent Error

### ISSUE THREAD

**Issue by Evansyjord** [BG-Software-LLC/SuperiorSkyblock2#695]
### Minecraft's Version

PaperSpigot-1.17.1 # 353

### Plugin's Version

Both SuperiorSkyblock2 v1.8 & SuperiorSkyblock2 v1.8 dev 338 

### Describe the bug

When you have a cobblestone generator and you mine an ore, in between the water and lava where the block is suppose to generate it acts as an air block and no block will regen until you update the lava source.

### To Reproduce

To reproduce this create 2 cobblestone generators using water logged stairs in an L shape as such:
After some time (depending on a certain tick) the lava block will "freeze" and not produce anymore ores

w = water logged stair
a = air block
l = lava

W
A
L A W

### Additional Information

Here is the error:
[16:47:53] [Server thread/ERROR]: Could not pass event BlockFromToEvent to SuperiorSkyblock2 v1.8
java.lang.NullPointerException: Cannot invoke "net.minecraft.world.level.block.Block.getBlockData()" because the return value of "org.bukkit.craftbukkit.v1_17_R1.util.CraftMagicNumbers.getBlock(org.bukkit.Material)" is null
	at com.bgsoftware.superiorskyblock.nms.v1_17_R1.NMSAlgorithmsImpl.getCombinedId(NMSAlgorithmsImpl.java:49) ~[SuperiorSkyblock2-1.8.jar:?]
	at com.bgsoftware.superiorskyblock.nms.v1_17_R1.NMSWorldImpl.setBlock(NMSWorldImpl.java:189) ~[SuperiorSkyblock2-1.8.jar:?]
	at com.bgsoftware.superiorskyblock.module.generators.listeners.GeneratorsListener.onBlockFormEvent(GeneratorsListener.java:90) ~[SuperiorSkyblock2-1.8.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor364.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.17.1.jar:git-Paper-353]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.17.1.jar:git-Paper-353]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.17.1.jar:git-Paper-353]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.world.level.material.FlowingFluid.spreadToSides(FlowingFluid.java:186) ~[app:?]
	at net.minecraft.world.level.material.FlowingFluid.spread(FlowingFluid.java:157) ~[app:?]
	at net.minecraft.world.level.material.FlowingFluid.tick(FlowingFluid.java:506) ~[app:?]
	at net.minecraft.world.level.material.FluidState.tick(FluidState.java:77) ~[app:?]
	at net.minecraft.server.level.ServerLevel.tickLiquid(ServerLevel.java:1059) ~[app:?]
	at com.destroystokyo.paper.server.ticklist.PaperTickList.tick(PaperTickList.java:306) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:721) ~[app:?]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1624) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:490) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1480) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1279) ~[patched_1.17.1.jar:git-Paper-353]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:319) ~[patched_1.17.1.jar:git-Paper-353]
	at java.lang.Thread.run(Thread.java:831) ~[?:?]

**Comment by OmerBenGera** [comment]
Show me your generator rates. Use `/is admin show` and send a screenshot of the output.

**Comment by Evansyjord** [comment]
![gen-rates](https://user-images.githubusercontent.com/44951418/140386061-f84d652b-4afb-4f21-83ec-13bf8f700701.PNG)

**Comment by OmerBenGera** [comment]
Use latest dev build, I added a new debug message which shows what blocks the plugin tries to generate. Use `/is admin debug Action: Generate Block, Island: <your-name>` and wait for the plugin to throw an error.
When it occurs, look which block-type it tries to generate and let me know.

**Comment by Evansyjord** [comment]
> Use latest dev build, I added a new debug message which shows what blocks the plugin tries to generate. Use `/is admin debug Action: Generate Block, Island: <your-name>` and wait for the plugin to throw an error. When it occurs, look which block-type it tries to generate and let me know.

Hi, my hard drive ended up corrupting on my PC so I had to switch to a backup of all my files on my R610 cluster. After doing this I haven't had any problems with the Minecraft server. I have tried to recreate the issue and also switched to the new stable build of SS2 but the error and bug I was encountering hasn't appeared yet. Not too sure what the problem was and now might be working perfectly fine as I can not seem to create the issue anymore. Feel free the close the ticket but If I do end up encountering it again later down the line I will inform you. Thank you
